### DISCUSSION TITLE
Is there any way to detect invincibility mode spoofing

### DISCUSSION THREAD

**Discussion by iwanttodo1t** [YimMenu/YimMenu#3231] [Q&A]
Guys, do you have any way?

**Comment by Yimura** [comment]
Not without them possibly noticing that a modder is present in the lobby and that you are the one trying to check it.

The current detection is completely passive without interfering or interacting with other players. Otherwise we would've added it.

**Reply by iwanttodo1t** [reply]
I'm willing to use some proactive methods to detect cheaters to restore some of the fun to the game
I have a crazy idea
Attack everyone remotely for a random period of time to see who doesn't take damage😀

**Reply by lonelybud** [reply]
it is a stupid idea

**Reply by xesdoog** [reply]
> Attack everyone remotely for a random period of time to see who doesn't take damage😀

You can do that with Lua but I agree with lonelybud, it's not the brightest idea. You will be griefing innocent players and also, if a player is inside an interior like an apartment, a business, the casino etc... your check will return a false positive since the game makes them invincible inside said interiors.

**Reply by Rxann** [reply]
> > Attack everyone remotely for a random period of time to see who doesn't take damage😀
> 
> 
> 
> You can do that with Lua but I agree with lonelybud, it's not the brightest idea. You will be griefing innocent players and also, if a player is inside an interior like an apartment, a business, the casino etc... your check will return a false positive since the game makes them invincible inside said interiors.

I agree with xesdoog. All the things above are valid and therefore, this will likely not be a core feature. Feel free to make it in Lua though.

**Comment by iwanttodo1t** [comment]
Cheaters can be disguised, is there anything that cheaters can't disguise, or if they do, they can't cheat, can we take advantage of that

**Reply by Rxann** [reply]
As Yimura said above, if there was we would most likely already have it.

**Reply by iwanttodo1t** [reply]
OK, I understand, thanks for your contribution.

**Reply by Rxann** [reply]
> OK, I understand, thanks for your contribution.

Thank you and best of luck if you wish to try to implement it in lua. It shouldn't be that hard to do!
